packages/components/src/snackbar/README.md

```diff
@@ -0,0 +1,48 @@
+# Snackbar
+
+Use Snackbars to communicate low priority, non-interruptive messages to the user.
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
```

Comment:
I think it would be a good idea to start adding "Accessibility guidelines" to all these READMEs similar to dev and design. cc @audrasjb @LukePettway

Reply:
That's a great idea!